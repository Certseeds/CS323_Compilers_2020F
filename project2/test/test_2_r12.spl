int test_2_r12()
{
  float arr1[10];
  float arr2[10];
  float a = 1.1;
  int i = 0;
  while (i < 10)
  {
    arr1[i] = a;
    a = a * a;
  }
  i = 0;
  while (i < 10)
  {
    arr2[arr1[i]] = a;
  }
  return 0;
}
