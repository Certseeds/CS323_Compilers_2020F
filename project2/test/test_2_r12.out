Error type 12 at Line 15: indexing by non-integer
Error type 5 at Line 15: unmatching type on both sides of assignment
